lib/plugins/boluswizardpreview.js

```diff
@@ -57,9 +55,9 @@ function init() {
   bwp.checkNotifications = function checkNotifications (sbx) {
 
     var results = sbx.properties.bwp;
-    if (results == undefined) return;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330305/issues/source?bid=2452379&fileBranchId=2452987#l58)